htmd/qm/psi4.py

```diff
@@ -0,0 +1,244 @@
+# (c) 2015-2017 Acellera Ltd http://www.acellera.com
+# All Rights Reserved
+# Distributed under HTMD Software License Agreement
+# No redistribution in whole or part
+#
+import os
+import numpy as np
+
+from htmd.qm.base import QMBase, QMResult
+
+
+class Psi4(QMBase):
+    """
+    Class to set up and run QM calculations with Psi4
+
+    Capabilities
+    ------------
+    - Single-point energy calculations with HF and DFT.
+    - Electronic properties: dipole and quadrupole monents, Mulliken charges, ESP at given points
+    - Geometry optimization with/without dihedral restraints
+    - Can use various queuing systems and on AceCloud
+
+    Attributes
+    ----------
+    molecule : :class:`FFMolecule`
+        Molecule
+    multiplity : int, default = 1
+        Multiplicity of the molecule (1 = singlet, 2 = doublet, 3 = triplet, etc.)
+    theory : str, default = 'BLYP3'
+        Level of theory. A full list is at Psi4.THEORIES
+    basis : str, default = '6-31G*'
+        Basis set. A full list is at Psi4.BASIS_SETS
+    correction : str, default = None
```

Comment:
The default is actually a string 'none' and not the `None` object

Reply:
Fixed 9f231ac